packages/connect-finance/src/connect.ts

```diff
@@ -1,27 +1,33 @@
-import { createAppConnector, Network } from '@aragon/connect-core'
-import { IFinanceConnector } from './types'
+import { createAppConnector } from '@aragon/connect-core'
 import Finance from './models/Finance'
 import FinanceConnectorTheGraph, {
   subgraphUrlFromChainId,
 } from './thegraph/connector'
 
 type Config = {
```

Comment:
I guess in our case we might want to have a base that we would extend, but I also like the idea of keeping them separate, even if it require some repetition, because it makes it easy for connector authors to take one and use it as a bootstrap. What do you think?